crates/sqlexec/src/context/local.rs

```diff
@@ -120,9 +123,7 @@ impl LocalSessionContext {
         // The main difference here is creating a new catalog mutator without a
         // metastore client to "detach" ourselves from the metastore we're
         // connected to (likely running in-memory).
-        let vars = self
-            .get_session_vars()
-            .with_remote_session_id(client.session_id(), VarType::System);
+        let vars = self.get_session_vars();
```

Comment:
I think I should also update vars here with the db id from the client.

Reply:
Yeah I think that would be reasonable since the "local" database we were connected to is now no longer accessible.